commands/team_e2e_test.go

```diff
@@ -6,6 +6,7 @@ package commands
 import (
 	"github.com/spf13/cobra"
 
+	"github.com/mattermost/mattermost-server/v5/model"
 	"github.com/mattermost/mmctl/client"
 	"github.com/mattermost/mmctl/printer"
```

Comment:
```suggestion
	"github.com/spf13/cobra"

	"github.com/mattermost/mattermost-server/v5/model"
	
	"github.com/mattermost/mmctl/client"
	"github.com/mattermost/mmctl/printer"
```

The goimports lint is complaining. This should fix it.

Reply:
Ok, thanks!!